eradiate/thermoprops/afgl1986.py

```diff
@@ -25,7 +28,7 @@ def make_profile(
     model_id : {"us_standard", "midlatitude_summer", "midlatitude_winter", "subarctic_summer", "subarctic_winter", "tropical"}, default: "us_standard"
         Model identifier.
 
-    levels : quantity or array
+    levels : quantity, optional
```

Comment:
If both scalars and arrays are allowed, it would be nice to mention it.

Reply:
I changed the docstrings too fast yesterday, I'll clarify them.